### ISSUE TITLE
Tooltips associated with the Ribbon control are not hoverable

### ISSUE THREAD

**Issue by ChrisSires** [dotnet/wpf#931]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)
.NET Core SDK (reflecting any global.json):
 Version:   2.1.507
 Commit:    e8520940d7

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.17763
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.507\

Host (useful for support):
  Version: 2.1.11
  Commit:  d6a5616240

.NET Core SDKs installed:
  2.1.500 [C:\Program Files\dotnet\sdk]
  2.1.505 [C:\Program Files\dotnet\sdk]
  2.1.507 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

* Windows version: (`winver`)  Version 1809 (OS Build 17763.503)

* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes

 
 
Tooltips associated with the Ribbon control are not hoverable; it is not possible to move the mouse over the tooltip to keep the tooltip.

Expected Result:

Hoverable: Should be able to hover on the additional content(tooltip)​

Actual Result:

The tooltip is not Hoverable. Tooltip is dismissed on hovering to it.
 
 **Minimal repro:**
 ```
<Window x:Class="TestWPF.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800"
        Name="NameOfWindow">

    <Grid>
        <Ribbon>
            <RibbonMenuButton ToolTip="tooltip not hoverable" Label="Ribbon Menu Button" />
            <CheckBox Content="CheckBox" ToolTip="also not hoverable"/>
        </Ribbon>
    </Grid>
</Window>   
```

Note: I expect this is the behavior for many other controls as well; this appears to be a new MAS accessibility requirement.

**Comment by arpitmathur** [comment]
Duplicate of #1014
